tasks/test_e2e.yml

```diff
@@ -0,0 +1,61 @@
+- name: copy shell script that executes prometheus query for collectd metrics
+  copy:
+    src: "{{ item }}"
+    dest: "/tmp"
+    owner: root
+    group: root
+    mode: 0755
+  with_items:
+    ['collectd_prometheus_query.sh']
```

Comment:
Suggestion: instead of creating this script, copying it over, running it and then examining the files, could you execute each of the commands directly using the command module?
That way, there's less chance of the play failing due to unrelated (i.e. copy, running or writing the files) reasons. In running the script, any of the curl commands could fail, and you wouldn't know which or why.
If the curl commands are run directly, and one set of metrics was unavailable, then the others will still be run instead of seeing that the `collectd_prometheus_query.sh` script failed.